packages/now-cli/test/dev/fixtures/handle-filesystem-missing/package.json

```diff
@@ -3,6 +3,6 @@
     "build": "mkdir -p dist && echo hello > dist/index.txt"
   },
   "dependencies": {
-    "umi": "latest"
+    "umi": "3.2.1"
```

Comment:
I had no idea why this test has a dependency. That might be able to be removed 🤔

Reply:
It looks like @AndyBitz added it in #2630 to test an infinite loop so it must trigger the zero config routes to be applied.

Reply:
Good catch. Removed in f749a1c8ebb228f8949ef9c6b8d0e23c5f1851cc.

Reply:
What I meant was that it might actually be needed now that I saw #2630.

@AndyBitz Can you confirm if `umi` is needed or not for this test?

Reply:
It has to be in the package.json, the version doesn't matter as far as I remember

Reply:
@TooTallNate Lets revert f749a1c

Reply:
Reverted.